### DISCUSSION TITLE
API Endpoint

### DISCUSSION THREAD

**Discussion by 4e576rt8uh9ij9okp** [BOINC/boinc#5004] [Q&A] [answered]
Hello,
I'm working on an automated system and I wonder if there is an API Endpoint in BOINC client/server where I'm able to say to "start working" and "stop working".

I'm more of a Javascript/NodeJS/Python guy and I run a RPI4 8GB that I want to control PC.

Thanks for your time 👍

**Comment by davidpanderson** [comment, marked as answer]
See https://boinc.berkeley.edu/trac/wiki/GuiRpc
